kibana-reports/public/components/report_definitions/report_settings/report_settings.tsx

```diff
@@ -123,18 +124,34 @@ export function ReportSettings(props: ReportSettingProps) {
 
   const handleReportSource = (e: React.SetStateAction<string>) => {
     setReportSourceId(e);
+    let fromInContext = false;
+    if (window.location.href.includes('?')) {
```

Comment:
Is this always true that only in-context menu url has '?'

Reply:
Yes, unless the url is changed manually